apps/test-suite/TestUtils.js

```diff
@@ -26,6 +26,18 @@ function optionalRequire(requirer) {
   }
 }
 
+// Both Location and TaskManager test suites define tasks in TaskManager.
+// Since tasks can only be defined during initialization phase (not as a result
+// of calling some function when the application is running, but rather in global scope),
+// we need to trigger code execution of these modules here (not in `getTestModules`
+// which is called in one of the components).
+const LocationTestScreen = optionalRequire(() => require('./tests/Location'));
+const TaskManagerTestScreen = optionalRequire(() => require('./tests/TaskManager'));
+// I have a hunch that optionalRequire doesn't work when *not* in global scope
+// since I had to move Camera screen import here too to get rid of an error
+// caused by missing native module.
+const CameraTestScreen = optionalRequire(() => require('./tests/Camera'));
```

Comment:
I have a feeling that we should move all those `optionalRequire`s and maybe just have an array of them defined in the global scope?

Reply:
👍, but let's do it in another PR some other time!